src/shared/database/connection.ts

```diff
@@ -2,17 +2,17 @@ import { DataSource } from 'typeorm';
 import { mongoConfig } from '../configs';
 import { CoinListEntity, CryptoDataEntity, UserEntity, VerificationCodeEntity } from './entities';
 
-export const connectToDB = async () => {
-  await new DataSource({
-    type: 'mongodb',
-    username: 'root',
-    password: 'pass',
-    database: 'crypto-calculator',
-    authSource: 'admin',
-    port: 27017,
-    useUnifiedTopology: true,
-    useNewUrlParser: true,
-    synchronize: false,
-    entities: [UserEntity, VerificationCodeEntity, CoinListEntity, CryptoDataEntity],
-  }).initialize();
-};
+export const MyDataSource = new DataSource({
+  type: 'mongodb',
+  username: mongoConfig.username,
+  password: mongoConfig.pass,
+  database: 'crypto-calculator',
+  authSource: 'admin',
+  port: 27017,
+  useUnifiedTopology: true,
+  useNewUrlParser: true,
+  synchronize: false,
```

Comment:
This may not even work for Mongo. It may only work for relational databases because they need to synchronize their models with the DB tables. But Mongo doesn't have any models so maybe you can delete this.

P.S. And for relational DB you would want to set this this way:
`process.env.NODE_ENV === 'development' ? true : false`